stack-hoauth2-2.6.yaml

```diff
@@ -0,0 +1,11 @@
+resolver: nightly-2022-12-09
+
+extra-deps:
+  - hoauth2-2.6.0
+
+# hoauth2 needs upper-bounds relaxed for
+#
+# - memory-0.18.0
+# - text-2.0.1
+#
+allow-newer: true
```

Comment:
There's a key in newer Stack where you can say which dependencies in specific you are allowing, rather than just ignoring all bounds ever. I'd for sure use that here now that it's available.